docs/cli/edgedb_migration.rst

```diff
@@ -24,13 +24,69 @@ description. It's also possible to split the schema definition across
 multiple ``.esdl`` files. The migration tools will read all of them
 and treat them as a single SDL document.
 
+Here is typical file tree::
+
+    $ tree dbschema
+    dbschema
+    ├── default.esdl
+    └── migrations
+        ├── 00001.edgeql
+        ├── 00002.edgeql
+        ├── 00003.edgeql
+        ├── ....
+        └── 00019.edgeql
+
+In the tree:
+
+* ``dbschema/*.esdl`` is a set of files that contains current database schema
+* ``default.esdl`` is named after ``module default`` that is usually used for
+  initial application schema. You don't have to stick to single file per
+  module, but this is a good start.
+* ``dbschema/migrations/*.edgeql`` is a sequence of migration files that when
+  applied bring database schema to the latest version. These files are mostly
+  managed by command-line tools, but have to be commited to source code
+  repository, and occasionally may need to be edited manually (common example
+  is when merging a branch).
+
+
+.. _ref_cli_edgedb_migration_workflow:
+
+General Workflow
+================
+
+1. Create or edit schema in ``dbschema/*.esdl`` files
+2. Before applying schema, create a migration file:
+
+   .. code-block:: bash
+
+      edgedb create-migration
+
+   It runs through changes interactively and puts
+   ``dbschema/migrations/<revision-number>.edgeql`` file into the file system
+   (later you want to check it into a version control)
+
+3. To apply schema run:
+
+   .. code-block:: bash
+
+      edgedb migrate
+
+
+It's worth mentioning that after code checkout (e.g. ``git pull``) it makes
+sense to run ``edgedb migrate`` again. Server has the latest migration id
+recorded so will only apply new migrations.
+
+At any time ``edgedb show-status`` describes if there are any pending
```

Comment:
```suggestion
At any time ``edgedb show-status`` outputs a list of pending
```

Reply:
It doesn't output a list of pending schema changes, we don't have a way to do that yet. And it doesn't output a list off pending migrations too (what if there are 100?). It only shows status, which is a number of persnding migrations and first change that is going to be inserted with create migrations. But I don't think we want to describe it's output at this precision.